posts/Ippon-USA-offers-a-helping-hand-to-those-impacted-by-natural-disasters.md

```diff
@@ -0,0 +1,32 @@
+ 
```

Comment:
The top of the page should have the following kind of information: https://github.com/ippontech/blog-usa#metadata

---
authors:
- Danny Collins
tags:
- life-at-ippon
- culture
date: 
title: "Ippon USA Offers a Helping Hand to Those Impacted By Natural Disaster"
image: 
---